Added SHipping Box page under Fulfillment section

Amir-HC opened the pull request:
Added Shipping Box page under Fulfillment section

Amir-HC committed:
Rename user-manuals/orders/fulfillment/shipping-box.md to user-manuals/fulfillment/shipping-box.md

changed the folder for the shipping box page

Amir-HC committed:
Update SUMMARY.md

Redirected Shipping box page to correct location

Amir-HC committed:
Update shipping-box.md

added code block for button

Amir-HC committed:
Added Netsuite-HotWax Commerce Shipping method mapping page

Amir-HC committed:
updated summary file to add Netsuite Shipping method page

Amir-HC committed:
Added Inventory Update page for Import App

Amir-HC committed:
Updated Sumamry for Inventory upload in Import App

Amir-HC committed:
Added Import page information and link to Data Config page

Dhiraj1405 commented on deployment&configuration/netsuite/flows/shipping-method-mapping/readme.md:
```diff
@@ -0,0 +1,38 @@
+# Shipping Method Mapping between HotWax Commerce and NetSuite
+
+The shipping method mapping between HotWax Commerce and NetSuite ensures that the shipping methods selected in Shopify correspond accurately to their counterparts in both HotWax Commerce and NetSuite. HotWax Commerce facilitates the mapping of shipping methods chosen from Shopify, providing accurate information to NetSuite. This process establishes coherence across platforms, maintaining consiste ...
```
Check this document, why exactly do we need a [Shipping method mapping](https://docs.hotwax.co/krewe/flows/salesorders/shippingmethmapping)?

Dhiraj1405 commented on deployment&configuration/netsuite/flows/shipping-method-mapping/readme.md:
```diff
@@ -0,0 +1,38 @@
+
+## Creating a Shipping Method in HotWax Commerce:
+
+Shipping methods that are supported by the retailer needs to be created in HotWax Commerce to ensure that the shipping information is sent to the Shipping carrier. Shipping methods can be created by following these steps:
```
Should have some external system context, as the order will go to Netsuite and the Shipping method needs to be presented there

Dhiraj1405 commented on deployment&configuration/netsuite/flows/shipping-method-mapping/readme.md:
```diff
@@ -0,0 +1,38 @@
+1. Access NetSuite Integration Page: Go to `Settings` > `NetSuite Integration`.
+
+ {% hint style="info" %}
+   If the page is not visible, navigate to the `General Settings` page and click on `Add NetSuite Data` button at the top.
```
Is it ADD or Load, we can write a little more what exactly this button does.

Dhiraj1405 commented on user-manuals/data-manager/configurations.md:
```diff
@@ -65,6 +65,36 @@ Editing an existing Data Manager Configuration allows users to update configurat
 
+## Verify the Data flow of Data Configurations
+
+In HotWax Commerce, the significance of the Data Manager Configurations feature extends beyond the management of data flow. A crucial aspect of this tool lies in the Data Manager Logs, which play a pivotal role in ensuring the accuracy and completeness of data imported into the Order Management System (OMS). These logs provide essential insights into the status of imported data, allowing users to ...
```
Remove unnecessary adjectives and shorten this

Dhiraj1405 commented on user-manuals/data-manager/configurations.md:
```diff
@@ -65,6 +65,36 @@ Editing an existing Data Manager Configuration allows users to update configurat
+### Steps
+
+
+1. Search for the Desired Service:
```
Search is keyword based we should not bind it with service, user may not know the service name sometimes.

Dhiraj1405 commented on user-manuals/data-manager/configurations.md:
```diff
@@ -65,6 +65,36 @@ Editing an existing Data Manager Configuration allows users to update configurat
+2. Open the Import Data for the Service:
+
+- Identify the relevant service in the search results.
+- Click on the blue open icon located in front of the service name.
```
Open link Icon

Dhiraj1405 commented on user-manuals/data-manager/configurations.md:
```diff
@@ -65,6 +65,36 @@ Editing an existing Data Manager Configuration allows users to update configurat
+- Review the displayed information to check the status of the service.
+- Ensure that the status of the service is finished. You can view different data status types [here.](https://docs.hotwax.co/user-guides/workflow/data-manager/imports#data-manager-logs)
+
+**View Logs for Detailed Information:**
```
We need to add the point where one of the data failed, so we can read the failed record, and understand the issue in the failed record.

Dhiraj1405 commented on user-manuals/import-app/inventory.md:
```diff
@@ -0,0 +1,67 @@
+   - Locate the HotWax Commerce Import app in the app launchpad. Log in with your user credentials. 
+
+   {% hint style="info" %}
+Only users with Super or Admin user permissions can log in to the Import App
```
Inventory section is restricted to Super and Admin

Merchandisers have access to Purchase Order page.

Dhiraj1405 commented on user-manuals/import-app/inventory.md:
```diff
@@ -0,0 +1,67 @@
+   
+     | Field               | Description                                                |
+     |---------------------|------------------------------------------------------------|
+     | Shopify Product SKU | The unique product identifier used by Shopify.          |
```
we should write that this can be changed as the user might not always have SKU configured.

Dhiraj1405 commented on user-manuals/import-app/inventory.md:
```diff
@@ -0,0 +1,67 @@
+     | Facility ID         | Identifies the specific facility associated with the product.|
+   
+   {% hint style="info" %}
+Choose your preferred product identifier by selecting the `Shopify Product SKU` field and choosing from the options in the dropdown menu.
```
We should convey that this is a default selection and can be changed to two other values. Also write Shopify barcode essentially mean UPCA.

Dhiraj1405 commented on user-manuals/import-app/inventory.md:
```diff
@@ -0,0 +1,67 @@
+Please ensure that all the products are synchronized in HotWax Commerce from ecommerce. If a product does not exist in HotWax Commerce, the corresponding records will not be processed.
+{% endhint %}
+
+7. **Handling Missing Facilities:**
```
Append `in imported data or CSV`

Dhiraj1405 commented on user-manuals/import-app/inventory.md:
```diff
@@ -0,0 +1,67 @@
+{% endhint %}
+
+7. **Handling Missing Facilities:**
+   - Merchandisers sometime make errors in their CSV files when creating them. For example if the CSV file has "Time Square" as facility name instead of the "Times Square" it can cause eoor when updating inventory. If any missing facilities or errors are detected during the process, click on the `Missing Facilities`.
```
This paragraph has a typo

Dhiraj1405 commented on user-manuals/import-app/inventory.md:
```diff
@@ -0,0 +1,67 @@
+   - Find the product SKU by utilising the search bar. You can also search from products using product name or HotWax Commerce ID.
+   - Navigate to the `Product Details` page by clicking on the product.
+   - Click on the `View Inventory` button to open the `Product Inventory View` page of that product.
+   - Here you will be able to verify the inventory changes under the `Inventory Items` section.
```
Should we also say logs here which are available on the page?

Amir-HC committed:
Update readme.md

Updated Netsuite shipping method mapping page to add more information in the introduction and for the Load Netsuite Data button.

Amir-HC committed:
Update configurations.md

Updated configurations page to add more detail and improve language

Amir-HC committed:
Update inventory.md

Updated inventory page for Im port app to add more details and enhanced readability

Dhiraj1405 committed:
Update inventory.md

Dhiraj1405 committed:
Update configurations.md

Amir-HC committed:
Merge remote-tracking branch 'upstream/user-guides-pub' into user-guides-pub
